documentation/content/fr/books/handbook/virtualization/_index.adoc

```diff
@@ -508,4 +508,10 @@ Cette section n'est pas encore écrite.
 [[virtualization-host]]
```

Comment:
In readiness for future translation, and for consistency with recent SEO-related <https://github.com/freebsd/freebsd-doc/commit/45b85d3fc587b0984b2b8878510429c214945114#diff-c7bbc24fc6b3cf53cb5fbb6d0be16159c70e0350dfc66ff72b95c05c44201fcdR11-R22>, maybe add, near the head of the page, below the `path` line: 

```text
aliases: [
"/fr/books/handbook/virtualization-guest-parallels/",
"/fr/books/handbook/virtualization-guest-virtualpc/",
"/fr/books/handbook/virtualization-guest-vmware/",
"/fr/books/handbook/virtualization-guest-virtualbox/",
"/fr/books/handbook/virtualization-host-virtualbox/",
"/fr/books/handbook/virtualization-host-bhyve/",
"/fr/books/handbook/virtualization-host-xen/"
]
```